### ISSUE TITLE
"Create your own schedule"

### ISSUE THREAD

**Issue by juand-r** [Mini-Conf/Mini-Conf#83]
I really liked the option to easily create your own schedule that was used at NAACL 2019 (https://naacl2019.org/schedule/).  It would be useful if it could also be organized in various ways (e.g., by date and time, or by track) and exported.

It would save time for people who are (a) trying to figure out which papers to attend given their interests, time zones, and other constraints, and then quickly look up when to log-in to each paper's QA session, and (b) perhaps be the starting point for a user to write notes on the conference (e.g., something like what I tried to do here: https://github.com/juand-r/ICLR-2020 , although that is very incomplete).

**Comment by hao-fang** [comment]
Our ACL2020 uses this code
https://github.com/acl-org/acl-2020-virtual-conference/blob/master/templates/components.html#L418-L506

It looks like this
![image](https://user-images.githubusercontent.com/9957482/87090795-64c0fa80-c1ed-11ea-8abf-6149c44b6fbf.png)

The js code is from this repo https://github.com/commonpike/add-to-calendar-buttons/blob/master/add-to-calendar.js with some modifications (see https://github.com/acl-org/acl-2020-virtual-conference/pull/218#discussion_r445352742) because we couldn't get the pop-up window working properly.

But I also like the NAACL2019 build-own-schedule feature. Not sure if it can automatically detect the user's own timezone?

**Comment by juand-r** [comment]
I think it was great that the virtual ACL2020 was able to display the time in the user's own timezone. And one could add the papers to a Google (or Outlook, etc..) calendar.  But I think there would be more flexibility in being able to do this in the virtual ACL2020 itself, since then one could potentially also sort the papers by both time and by track (topic). I noticed it already highlights which papers were visited (which was also a nice feature),

**Comment by srush** [comment]
Yeah! I think this would be a super cool feature. It's hard though. Have to think through the best way to execute it.

**Comment by yarikoptic** [comment]
Possibly a crazy idea (sorry -- I have no idea of mini-conf internals, yet at least): wouldn't it be possible at the client level to "link back" personal (google, whatnot) .ical into mini-conf? If events (papers) contain not only the date/time slot but also link to the actual event/paper - then mini-conf interface could display "personal schedule" as well (re-fetching if updated), thus requiring no need for server-side support.
